core/const.py

```diff
@@ -110,6 +112,229 @@ class ITLRegion(StrEnum):
     SouthWest = "TLK"
 
 
+class OutputCategories(StrEnum):
+    NUMBER_OF_ADDITIONAL_ENTERPRISES_WITH_BROADBAND_ACCESS_OF_AT_LEAST_30MBPS = (
```

Comment:
I may be missing something, but these are not actually Outcome Categories, but Output names?